pallet-staking/src/benchmarking.rs

```diff
@@ -816,7 +820,7 @@ mod tests {
                     n,
                     <Test as Config>::MaxNominatorRewardedPerValidator::get() as u32,
                     false,
-                    RewardDestination::Staked,
+                    RewardDestination::Stash,
```

Comment:
Why did this get changed? it seems unrelated to the other changes

Reply:
Unrelated change indeed